tests/robustness/robustness_test/main_test.go

```diff
@@ -62,6 +62,9 @@ func TestMain(m *testing.M) {
 
 		prev := rs.ContentFormat.MutableParameters.Version
 
+		// This variable is reset in cleanup function.
+		os.Setenv("KOPIA_UPGRADE_LOCK_ENABLED", "1")
+
 		log.Println("Old repository format:", prev)
 		th.snapshotter.UpgradeRepository()
```

Comment:
Nit: the success or failure of this function is now contingent on doing some work outside it. Is this intended? Could that be confusing in the future, say if someone wants to reuse this function?

Reply:
Moved the setEnv call to the method and added the reset to cleanup function.